pkg/controller/controller.go

```diff
@@ -96,13 +96,7 @@ func (c *Controller) CreateVolume(contx context.Context, opt *pb.CreateVolumeOpt
 	log.Info("Controller server receive create volume request, vr =", opt)
 
 	ctx := osdsCtx.NewContextFromJson(opt.GetContext())
-	if opt.ProfileId == "" {
-		log.Warning("Use default profile when user doesn't specify profile.")
-		prf, err = db.C.GetDefaultProfile(ctx)
-		opt.ProfileId = prf.Id
-	} else {
-		prf, err = db.C.GetProfile(ctx, opt.ProfileId)
-	}
+	prf, err = db.C.GetProfile(ctx, opt.ProfileId)
```

Comment:
I think this can be directly decoded from `opt.Profile`.

Reply:
good idea